unit1-intro-to-programming-in-javascript/hoisting-exercises.md

```diff
@@ -54,7 +68,66 @@ for (var i = 0; i < 5; i++) {
 }
 
 console.log("Test: " + test)
+```
 
 
-#### 4
+#### 5
+```js
+var test = "initial value"
+var f = function () {
+    test = "new value"
+
+    if (test) {
+        test += " and more!"
+    } else {
+        test = "It begins here"
+    }
+    g()
+}
+var g=function(){
+	var test="newnew value"
+}
+f()
+console.log("Test: " + test)
+```
+
+#### 6
+```js
+var test = "initial value"
+var f = function () {
+    test = "new value"
+
+    if (test) {
+        test += " and more!"
+    } else {
+        var test = "It begins here"
+    }
+    g()
+}
+var g=function(){
+	test="newnew value"
+}
+f()
+console.log("Test: " + test)
+```
+
+#### 7
+```js
+var test = "initial value"
+var f = function () {
+	g()
```

Comment:
INDENTATION!!! Make sure you update your text editor to use SPACES instead of tabs. Make it consistent with the rest of the file (4 spaces per tab)